css/style.css

```diff
@@ -66,6 +71,7 @@ a:hover {
     margin: 50px auto;
     font-size: 2rem;
     color: rgba(255,255,255,1);
```

Comment:
Unless you're using transparency (the last number in the `rgba()` function) you should probably use hex (`#ffffff`). Please switch `rgba(255,255,255,1)` to `#ffffff` here and everywhere else (Option+3 on your keyboard to get the # symbol). Don't change it in places where the colour is transparent.

Reply:
Mike always say we should be consistent , there for I was scared using two different way of expression colors

Reply:
I had that internal dilemma, too, so I asked Mike 👍 
<img width="353" alt="Screenshot 2021-06-03 at 14 44 29" src="https://user-images.githubusercontent.com/14294127/120655090-363db500-c47a-11eb-9473-6256a3834338.png">

Reply:
Mike is inconsistent

Reply:
ahah thanks for clarification :)